src/utils.ts

```diff
@@ -8,29 +8,35 @@ import { parse as urlParse} from 'url';
 
 export const entities = new Entities();
 
-const downloadGetStream = (url, data?) => needle.get(url, data);
+const downloadGetStream = (url, options?) => {
+  let defaultOptions = {
+    'follow': 10,
+  };
+  let mergedOptions = Object.assign(defaultOptions, options);
+  needle.get(url, mergedOptions);
```

Comment:
@thomas-profitt You should put a `return` on this line to return whatever `needle.get` returns back to the caller. The `=>` shorthand automatically returns for you, so your code isn't 100% equivalent unless you `return` like it would have.

Reply:
Whoops. Ruby/Elixir habits will be the death of me. At least I'm not missing any parentheses.